django/db/backends/oracle/features.py

```diff
@@ -208,3 +208,7 @@ def supports_aggregation_over_interval_types(self):
     @cached_property
     def bare_select_suffix(self):
         return "" if self.connection.oracle_version >= (23,) else " FROM DUAL"
+
+    @cached_property
+    def supports_partial_json_update(self):
+        return self.connection.oracle_version >= (21,)
```

Comment:
This seems correct according to https://apex.oracle.com/database-features/

However, the function seems to be documented for 19c as well: https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/json_transform.html

According to https://stackoverflow.com/a/75286939 it was backported to 19c in 19.10. Could we update this and see if it works? According to https://code.djangoproject.com/wiki/CI we use v19.3.0.0.0 on the CI though, so that may need to be upgraded, or we can test ourselves.

Reply:
We agreed on following the https://apex.oracle.com/database-features/ website.